_posts/national/united-kingdom/united-kingdom.Rmd

```diff
@@ -36,8 +36,7 @@ source(here::here("utils/subnational_defaults.R"), local = TRUE)
 widget
 ```
 
-<br>
-`r widget_caption`
+<br> `r widget_caption`
```

Comment:
There is an authorship section in the YAML. I suggest updating to that used in the paper. There is also a citation section where the paper can be linked. Check out the docs for details (https://rstudio.github.io/distill/citations.html)

Reply:
I'd also suggest mentioned where the methodology for this is outlined (i.e your paper) at the top somewhere.